ci/docker-run.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# This is intended to be run inside the docker container as the command of the docker-compose.
+set -ex
+if [ "$INTEGRATION" == "true" ]; then
```

Comment:
Would it be better to have and use a separate `docker-compose.yml` for integration tests, since non-integration tests cherry-pick a single service to run out of the composition?